### ISSUE TITLE
`Method to resolve Unstoppable Domains` dropdown items are not translated in other locales

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-browser#14862]
## Description 

Method to resolve Unstoppable Domains dropdown items are not translated in other locales

## Steps to Reproduce

   1. Set OS locale to France or any other locale
   2. Open brave 1.22.66
   3. Open brave://settings/extensions
   4. `Method to resolve Unstoppable Domains` dropdown items are not translated in other locales

## Actual result:

![image](https://user-images.githubusercontent.com/38657976/111980259-0b56ec80-8b2c-11eb-8fe9-dd1553ade73e.png)

## Expected result:

Dropdown items should be translated
## Reproduces how often: 

Easy

## Brave version (brave://version info)

Brave | 1.22.66 Chromium: 89.0.4389.90 (Build officiel) (64 bits)
-- | --
Révision | 62eb262cdaae9ef819aadd778193781455ec7a49-refs/branch-heads/4389@{#1534}
Système d'exploitation | Windows 10 OS Version 2004 (Build 19041.867)

## Version/Channel Information:

- Can you reproduce this issue with the current release? NA
- Can you reproduce this issue with the beta channel? NA
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

## Miscellaneous Information:

cc: @brave/legacy_qa @rebron @mkarolin

**Comment by mkarolin** [comment]
cc: @yrliou

**Comment by LaurenWags** [comment]
Reproduces on Nightly as well

**Comment by kjozwiak** [comment]
Added `QA/Blocked` as we'll need to wait for the translations to get merged into `1.23.x`. @mkarolin please keep us updated re: merging translations into `1.23.x` and unblocking the above 👍

**Comment by stephendonner** [comment]
Removing `QA/Blocked` label as we now have the `1.23.70` build.

Verified `PASSED` using 

Brave | 1.23.70 Chromium: 90.0.4430.70 (Official Build) (x86_64)
-- | --
Revision | 3954de7175366f3b7edca576f140dfa273e6b5ae-refs/branch-heads/4430@{#1210}
OS | macOS Version 11.2.3 (Build 20D91)

Confirmed using the inline steps to reproduce that the dropdown items are now correctly translated in both French and Spanish.

French | Spanish
-------|----------
<img width="844" alt="Capture d’écran 2021-04-13 à 15 00 41" src="https://user-images.githubusercontent.com/387249/114627398-0d632580-9c6a-11eb-9dfb-b13f29a1aedd.png"> | <img width="847" alt="Captura de pantalla 2021-04-13 a las 15 03 08" src="https://user-images.githubusercontent.com/387249/114627401-0e945280-9c6a-11eb-868e-1ffdc6b3d8c4.png">
